cms/static/sass/assets/_fonts.scss

```diff
@@ -3,9 +3,9 @@
 @font-face {
     font-family: 'Open Sans';
     src:
-        url('../fonts/OpenSans/OpenSans-Light-webfont.woff2') format('woff2'),
-        url('../fonts/OpenSans/OpenSans-Light-webfont.woff') format('woff'),
-        url('../fonts/OpenSans/OpenSans-Light-webfont.ttf') format('truetype');
+        url($static-path + '/fonts/OpenSans/OpenSans-Light-webfont.woff2') format('woff2'),
```

Comment:
Hmmm, would this better to handle with the following syntax (to account for Sass interpolation)?

```scss
url('#{$static-path}/fonts/OpenSans/OpenSans-Light-webfont.woff2') ...
```

It would simplify the structure of the string here. Also, we ran into some issues in the UXPL's Sass compilation without doing this. Not sure how you guys are avoiding it. For more info on Sass interpolation, this was a good read - http://webdesign.tutsplus.com/tutorials/all-you-ever-need-to-know-about-sass-interpolation--cms-21375